pulpcore/cli/rpm/distribution.py

```diff
@@ -67,6 +67,15 @@ def distribution(ctx: click.Context, pulp_ctx: PulpCLIContext, distribution_type
             "auto-distribute."
         ),
     ),
+    click.option(
+        "--generate-repo-config",
+        type=click.BOOL,
+        default=False,
+        help=_(
+            "Option specifying whether ``*.repo`` files will be generated and served."
+            " Defaults to False."
+        ),
+    ),
```

Comment:
This flag will only appear in pulp_rpm 3.23, right? You should make it a `pulp_option` with `needs_plugins=[PluginRequirement("rpm", specifier=">=3.23.dev")]`.

Reply:
thanks for the tip